index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Isaac Jung's Resume</title>
+    <link rel='stylesheet' href='styles.css' />
+</head>
+
+<body>
+    <div class="page u-clearfix">
+        <main>
+            <div class="content01">
+                <h1>Graphic Designer & Sound Artist</h1>
+                <p><u>Isaac Jung</u> is based in Seoul, Republic of Korea.
+                    A flexible creator skilled in conceptualizing visual systems, analyzing product UIUX,
+                    sound editing, and site specific sonic installations He is also a traveler,
+                    an avid barista, a part-time composer, a devoted table tennis player,
+                    and an eager learner in all the arts.</p>
+            </div>
+            <div class="content02">
```

Comment:
Going forward, I would challenge you to have more meaningful and easily discernible class names.